### ISSUE TITLE
Arguments filed is not working as expected in Azure Pipeline Extension

### ISSUE THREAD

**Issue by durgakhumbu** [rdagumampan/yuniql#73]
Hi, 

We set target version arguments but it is throwing error. Please find the below log. If I remove the value in "Arguments" field then it is working as expected. 

C:\hostedtoolcache\windows\yuniql\0.0.0\x64\yuniql.exe run -p D:\a\1\s\custom-direct\database\root -c Server=******.us-east-1.rds.amazonaws.com;Database=*******;Uid=admin;Pwd=******** --platform mysql --delimiter , **"--target-version 'v0.00'"**

yuniql 0.355.0
Copyright 2019 Rodel E. Dagumampan

ERROR(S):
  Option 'target-version 'v0.00'' is unknown.

**Comment by rdagumampan** [comment]
@durgakhumbu , thanks for you reaching out. I hope I understand the case right. 

At a minimum, yuniql requires that the scripts are organized in a certain structure. See an example here: https://github.com/rdagumampan/yuniql/tree/master/samples/basic-mysql-sample To initiate the structure, you can download the CLI via https://yuniql.io/docs/install-yuniql/ and call `yuniql init`

This example below generate these CLI commands:

```console
[command]C:\hostedtoolcache\windows\yuniql\0.355.0\x64\yuniql.exe run -p D:\a\r1\a\_rdagumampan_yuniql\samples\basic-mysql-sample -c "***" --platform mysql -t v0.00 -k VwColumnPrefix1=Vw1,VwColumnPrefix2=Vw2,VwColumnPrefix3=Vw3,VwColumnPrefix4=Vw4 --delimiter , --debug
```

This is a sample YAML:
```yaml
#Your build pipeline references a secret variable named ‘AzMySqlDemoDatabase’. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab, and then select the option to make it secret. See https://go.microsoft.com/fwlink/?linkid=865972

steps:
- task: rdagumampan.yuniql-azdevops-extensions.run-task.RunYuniqlCLI@0
  displayName: 'Run YUNIQL CLI'
  inputs:
    version: v0.355.0
    connectionString: '$(AzMySqlDemoDatabase)'
    workspacePath: '$(System.DefaultWorkingDirectory)\_rdagumampan_yuniql\samples\basic-mysql-sample'
    targetPlatform: mysql
    targetVersion: v0.00
    tokenKeyValuePair: 'VwColumnPrefix1=Vw1,VwColumnPrefix2=Vw2,VwColumnPrefix3=Vw3,VwColumnPrefix4=Vw4'
    additionalArguments: '--debug'
```

This is a sample Classic/UI

![image](https://user-images.githubusercontent.com/5895952/80099276-7d1a6500-856e-11ea-8543-d319c4ea6c7e.png)


HTH / Rodel

**Comment by rdagumampan** [comment]
@durgakhumbu I will close this for now. 
Just reach out or re-open if this remains an issue.

HTH/Rodel

**Comment by rdagumampan** [comment]
@durgakhumbu ,

I only get to understand this when another user reported the issue. Apologies for that. The fix is now part of latest release of Az DevOps Task.

FYI, I have just released v0.465.0 this morning. This is a BETA RELEASE and maybe have breaking changes on your pipelines. Please refer to this release note and create ticket if you found an issue.

https://github.com/rdagumampan/yuniql/releases/tag/v0.465.0

Thanks for your interest with yuniql.
Cheers, Rodel
